akka-persistence-query/src/main/scala/akka/persistence/query/javadsl/EventsByPersistenceIdQuery.scala

```diff
@@ -17,7 +17,8 @@ trait EventsByPersistenceIdQuery extends ReadJournal {
    * Query events for a specific `PersistentActor` identified by `persistenceId`.
    *
    * You can retrieve a subset of all events by specifying `fromSequenceNr` and `toSequenceNr`
-   * or use `0L` and `Long.MaxValue` respectively to retrieve all events.
+   * or use `0L` and `Long.MAX_VALUE` respectively to retrieve all events. The query will
```

Comment:
Makes sense given it is a javadsl, even though the [LeveldbReadJournal.scala implementation uses Long.MaxValue](https://github.com/akka/akka/blob/master/akka-persistence-query/src/main/scala/akka/persistence/query/journal/leveldb/scaladsl/LeveldbReadJournal.scala#L114)

Reply:
This matches implementation and documentation in Cassandra and Leveldb plugin